src/main/java/com/metamx/metrics/JvmMonitor.java

```diff
@@ -47,7 +47,7 @@ public boolean doMonitor(ServiceEmitter emitter)
       final String kind = entry.getKey();
```

Comment:
but this PR renames dimensions, not metric names, so that's cool, the doc update really concerns renaming metric names and doesn't yet have the list of dimensions